frontend/components/panels/NetProfitCheckPanel.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+    <BasePanel :current-state="currentStateAndTitle.name" :disabled="!isExplanationsShown">
+        <template #title> {{ currentStateAndTitle.title }} </template>
+        <TextBlock v-if="isExplanationsShown">
+            Executing an auction with a negative net profit is possible, but according to our calculations it will
+            result in the net loss due to the transaction fees being higher then the gross profit.
```

Comment:
- [ ] let's write "a net loss" instead of "the net loss"
- [ ] "higher than" instead of "higher then"